client/src/AppBundle/Controller/Admin/Client/ReportController.php

```diff
@@ -438,6 +452,35 @@ private function populateReportFromSession(Report $report, array $sessionData):
         }
     }
 
+    /**
+     * @Route("manage-close-report-confirm", name="admin_report_manage_close_report_confirm")
+     * @Security("has_role('ROLE_ADMIN') or has_role('ROLE_AD')")
+     *
+     * @param Request $request
+     * @param $id
+     * @return array|RedirectResponse
+     * @Template("AppBundle:Admin/Client/Report:manageCloseReportConfirm.html.twig")
+     *
+     * @throws \Exception
+     */
+    public function manageCloseReportConfirmAction(Request $request, $id)
+    {
+        $form = $this->createForm(CloseReportConfirmType::class);
```

Comment:
For a form this simple, you might consider [building it on the fly](https://symfony.com/doc/current/forms.html#creating-forms-in-controllers) rather than having a dedicated class
```suggestion
        $form = $this->createFormBuilder()
            ->add('save', SubmitType::class)
            ->getForm();
```

Reply:
I considered this, but decided against the inconsistency. I quite like the predictability that we currently have: if there's a form, there's a class. Admittedly it's a tad bloated but there is always a trade off - and makes it easier to add translations and validation rules if we need to.

Reply:
Fair enough, I see how it's helpful for translations